### PR TITLE
Change versionning warning

### PR DESCRIPTION
**Pull request by bidoubiwa**
I removed the version in the warning as it was not updated between updates of MeiliSearch. Instead I explained how to know which MeiliSearch instance if represented in the documentation.

### CODE REVIEW

File: references/README.md [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
+You might find the answers to some of your questions [in the FAQ](/faq/faq.md) 💡
 
 :::
 
-::: tip
+## Version Compatibility
 
-You might find the answers to some of your questions [in the FAQ](/faq/faq.md) 💡
+The documentation of MeiliSearch is compatible with the latest version of MeiliSearch. We do not provide multi-versionning documentation.
+If you have a doubt concerning the version of [MeiliSearch](https://github.com/meilisearch/MeiliSearch), it is written on the upper navbar on the left side.
```
Comment (CaroFG) [inline references/README.md:14]: "If you have a doubt about the version of Meilisearch" :  I find  "a doubt" very wide, I could have a doubt about the differences between the version I have installed and the latest one.  Or maybe I don't know which version I have installed in my computer. Maybe just change this to something like : "The actual version of Meilisearch is written..."

**Review by CaroFG — DISMISSED** [review]

File: references/README.md [hunk 2]
```diff
@@ -2,17 +2,18 @@
 
 Welcome to the MeiliSearch API documentation.
 
-::: warning
+::: tip
 
-The documentation is written for the latest stable release: [v0.16.0](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.15.0).
+You might find the answers to some of your questions [in the FAQ](/faq/faq.md) 💡
```
Comment (react-learner) [inline references/README.md:7]:
```suggestion
Check out [the FAQ](/faq/faq.md) for answers to some common questions 💡
```

File: references/README.md [hunk 3]
```diff
@@ -2,17 +2,18 @@
...
 
 :::
 
-::: tip
+## Version Compatibility
 
-You might find the answers to some of your questions [in the FAQ](/faq/faq.md) 💡
+The documentation is compatible with the latest version of MeiliSearch. We do not provide multi-versionning documentation.
```
Comment (react-learner) [inline references/README.md:13]:
```suggestion
This documentation reflects only the latest version of MeiliSearch. We do not provide multi-version documentation.
```

File: references/README.md [hunk 4]
```diff
@@ -2,17 +2,18 @@
...
+The actual version of [MeiliSearch](https://github.com/meilisearch/MeiliSearch) is written on the upper navbar on the left side.
```
Comment (react-learner) [inline references/README.md:14]:
```suggestion
The current version of [MeiliSearch](https://github.com/meilisearch/MeiliSearch) can be found on the left side of the upper navbar.
```

**Review by react-learner — CHANGES_REQUESTED** [review]

**Pull request merged**